src/include/iscsi-lio-server/widgets.rb

```diff
@@ -226,26 +226,26 @@ def AuthTerm(discovery)
           CheckBox(
             Id(:auth_in),
             Opt(:notify),
-            _("Incoming Authentication"),
+            _("Authentication by Targets"),
             false
           )
         ),
         HBox(
-          InputField(Id(:user_in), Opt(:hstretch), _("Username")),
+          InputField(Id(:user_in), Opt(:hstretch), _("UserID")),
           Password(Id(:pass_in), _("Password"))
         ),
         VSpacing(1.5),
         Left(
           CheckBox(
             Id(:auth_out),
             Opt(:notify),
-            _("Outgoing Authentication"),
+            _("Authentication by Initiators"),
             false
           )
         ),
         HBox(
-          InputField(Id(:user_out), Opt(:hstretch), _("Username")),
-          Password(Id(:pass_out), _("Password"))
+          InputField(Id(:user_out), Opt(:hstretch), _("Userid_Mutual")),
```

Comment:
I removed muaual and "_", I believe just "UserID" and "Password" are fine, because "Authentication by Initaitors" and "Authentication by Targets" already describe the directions well.